### PR TITLE
Added CMD tests

### PR DESCRIPTION
**Pull request by mikiquantum**
Issue Link: <Github link>

### CODE REVIEW

**Comment by vedhavyas** [comment]
@mikiquantum Amazing!!
Can we use go tests instead of shell scripts? Useful for debugging

**Comment by mikiquantum** [comment]
@vedhavyas I want to call the binary explicitly (such as functional test) as a black box test, I can convert the shell script into a go file, but I am still interested on invoking the cmd binary. 
In any case there will not be a debugging option, for that we have (or should have) unit/integration testing.

**Comment by mikiquantum** [comment]
@vedhavyas I actually moved it to a go file, mainly because I noticed I was duplicating the way that we read smart contract addresses.

File: cmd/centrifuge_cmd_test.go [hunk 1]
```diff
@@ -0,0 +1,38 @@
+// +build cmd
```
Comment (vedhavyas) [inline cmd/centrifuge_cmd_test.go]: Any specific reason why new build tag instead of unit test. I think these belong to unit test IMO

**Comment by mikiquantum** [comment]
@vedhavyas this test suite requires an ethereum node running, that belongs to the integration tests domain. On the other hand, this is supposed to be a functional one as there is no access to code base during the test, so that is why I decided to create own tag.

**Pull request merged**